docs/uab_cloud/project_space_mgmt.md

```diff
@@ -0,0 +1,82 @@
+
+![!RC Dashboard Screenshot showing Project Space Name](images/rc_proj_dashboard.png)
+
+When in the project space, select `Compute` and then `Instances` to see available instances in your project space.
+
+<!-- markdownlint-disable MD046 -->
+!!! note
+
+    Please note, all members of your project space, can create, and delete instances. In essence, all members of the project space, have the same user privileges.
+<!-- markdownlint-enable MD046 -->
+
+## Sharing an Instance in a Project Space
+
+There are a couple of ways to collaborate while using on an instance, this article would focus on some easy-to-use ways to use an instance in a shared project space.
+
+### Creating an Image From an Instance Using Snapshots
+
+Snapshots in OpenStack are used like Images, thereby making it relatively easy to create Images from an Instance. To create an Image that can then be shared with other members of your project space, so they can replicate your Instance, please follow the below steps;
+
+1. Go to the `Compute` pane and then navigate to the `Instances` tab in your project space dashboard. You will then see a list of available instances.
+
+    ![!Screenshot showing Compute and Instance Buttons](images/rc_comp_instance.png)
+
+1. Select the Instance you want to create an Image for. Under the Actions, click on `Create Snapshot`.
+
+    ![!Create Snapshot Button](images/rc_snapshot_button.png)
+
+1. Insert a name for the Snapshot, and then click `Create Snapshot`.
+
+    ![!Menu Box to create Snapshot](images/rc_instance_snapshot.png)
+
+1. You will be redirected to the `Images` page, where you newly created Image will appear amongst a list of other available images.
+
+    ![!Screenshot showing created Images](images/rc_created_image.png)
+
+The created Image can then be launched, following the same instructions for creating an [Instance](../uab_cloud/tutorial/instances.md). This method would be most ideal if you want to recreate an environment for performing an analysis, but would prefer the workflow be run on different VMs, or to separate datasets or create some form of access restriction on particular research.
+
+### Creating an Image From a Volume
+
+There are detailed instructions here on how to create an image from a Volume, [here](snapshots.md#creating-a-volume-snapshot).
+
+### Using a Key Pair to SSH
+
+Another way to access a created instance would be to create a public key and private key for your local machine, and then share this public key with the creator of the instance. As creator of the instance you would need to add individual public keys of persons who you would want to access the created VM into the `authorized_keys` file. You can edit this file by using the command below, add the shared public key in a new line inside the file (copy and paste). Save the file and follow instructions here for remote accessing your instance using [`SSH`](remote_access.md).
```

Comment:
See previous note about adding multiple people to Key Pair.

Reply:
Noted